packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.base.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { IVerticalDividerProps, IVerticalDividerPropsStyles, IVerticalDividerStyles } from './VerticalDivider.types';
+import { classNamesFunction } from '../../Utilities';
+const getClassNames = classNamesFunction<IVerticalDividerPropsStyles, IVerticalDividerStyles>();
+
+export const VerticalDividerBase = (props: IVerticalDividerProps) => {
+  const { styles, theme, getClassNames: deprecatedGetClassNames } = props;
+  const classNames = getClassNames(styles, { theme: theme, getClassNames: deprecatedGetClassNames });
```

Comment:
you are not passing the `className` prop to the styles function. You are attaching the class to the `wrapper` style area but you are not passing anything in here.